Samples/Insights/cpp-win32/packages.config

```diff
@@ -0,0 +1,9 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<packages>
+  <package id="Microsoft.Windows.ImplementationLibrary" version="1.0.210803.1" targetFramework="native" />
+  <package id="Microsoft.WindowsAppSDK" version="1.0.0-experimental1" targetFramework="native" />
+  <package id="Microsoft.WindowsAppSDK.DWrite" version="1.0.0-experimental1" targetFramework="native" />
+  <package id="Microsoft.WindowsAppSDK.Foundation" version="1.0.0-experimental1" targetFramework="native" />
+  <package id="Microsoft.WindowsAppSDK.InteractiveExperiences" version="1.0.0-experimental1" targetFramework="native" />
+  <package id="Microsoft.WindowsAppSDK.WinUI" version="1.0.0-experimental1" targetFramework="native" />
+</packages>
```

Comment:
Would be good to use just the packages we need here.

Reply:
These packages come as a side-effect of adding WindowsAppSDK reference. These are not explicitly added.

Reply:
You're referencing the metapackage. That pulls in everything. You can reference just the Foundation package.